plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/connection/OAuthDetailView.java

```diff
@@ -145,11 +145,21 @@ public Composite createControls(Composite parent, Object context, DataBindingCon
 	public void onVisible(IObservableValue detailsViewModel, DataBindingContext dbc) {
 		bindWidgetsToInternalModel(dbc);
 		rememberTokenCheckbox.setText("&Save token (could trigger secure storage login)");
+		setCheckboxVisible(true);
```

Comment:
I tried it and it works. It must be I did something else wrong when tried without update() and layout(true).